packages/client/src/app/services/storage.service.js

```diff
@@ -0,0 +1,42 @@
+export class StorageService {
```

Comment:
This class adds no functionality and you can just use local storage directly

Reply:
Removed the StorageService